Common/include/CConfig.hpp

```diff
@@ -5281,7 +5428,12 @@ class CConfig {
   /*!
    * \brief Indicates whether electron gas is present in the gas mixture.
    */
-  bool GetIonization(void);
+  bool GetIonization(void) const { return ionization; }
+
+  /*!
+   * \brief Indicates whether electron gas is present in the gas mixture.
```

Comment:
This doc string hasn't been updated.

Reply:
not sure if I understand what you mean

Reply:
@WallyMaier pls clarify

Reply:
Sorry.  The brief for GetFrozen is the same as GetIonization